samples/subsys/fs/littlefs/sample.yaml

```diff
@@ -52,7 +52,7 @@ tests:
     build_only: true
     platform_allow: stm32h747i_disco/stm32h747xx/m7
     extra_args:
-      - EXTRA_CONF_FILE=boards/stm32_blk.conf
+      - EXTRA_CONF_FILE=overlay-stm32_blk.conf
       - CONF_FILE=prj_blk.conf
```

Comment:
No, do NOT do this

Reply:
@thedjnK the majority of overlay config files already have the prefix `overlay-`, this change just makes things consistent & makes it less likely for unfamiliar users to confuse base config files with overlay ones.

Reply:
Overlay isn't even right, overlay is a devicetree term, it has no relation to Kconfig at all

Reply:
We can debate the terminology, but it's still an `overlay` file that is applied on top of the base one & overrides it.
We can replace with `extra` if that's better.

Reply:
It's not an overlay, we will do no replacing of sorts

Reply:
> No, do NOT do this

@thedjnK I moved that file because it applies to multiple boards & thus does not belong in `boards`.

Reply:
> Then why did you add it wrong in the first place?

I was new to Zephyr :)

> it makes no sense to break everyone's workflow in existing samples which use those names, but new samples/tests should not

I agree that the renaming is likely to cause breakage out-of-tree... will remove, thanks for your review!